.github/workflows/test.yml

```diff
@@ -28,6 +28,11 @@ jobs:
       # https://twitter.com/jonhoo/status/1571290371124260865
       - name: cargo test --locked
         run: cargo test --locked --all-features --all-targets
+      - name: cargo test network config
```

Comment:
I don't think we need this test here if we have a unit-test that parses `networks.json.example`